docs/3.0/cache.md

```diff
@@ -39,6 +39,49 @@ config.cache_store = ActiveSupport::Cache.lookup_store(:solid_cache_store)
 Our computed system do not use MemoryStore in production because it will not be shared between multiple processes (when using Puma).
 :::
 
+## Improving Performance
+
+Avo caches each record on the index view for improved performance. However, associated records may not be automatically updated when certain associations change. To prevent this:
+
+### Option 1: Use `touch: true` on association
+
+Example with Parent Model and Association
+```ruby
+  class Post < ApplicationRecord
+    has_many :comments, dependent: :destroy
+  end
+```
+Example with Child Model and Association with `touch: true`
+```ruby
+  class Comment < ApplicationRecord
+    belongs_to :post, touch: true
+  end
+```
+
+### Option 2: override `cache_hash` method on resource to take associations in consideration
+Avo, internally, uses the cache_hash method to compute the hash that will be remembered by the caching driver when displaying the rows.
+
+You can take control and override it on that particular resource to take the association into account.
```

Comment:
Let's reformulate a little bit and also mention this issue: https://discord.com/channels/740892036978442260/1234796408175853649